app/sitemap.ts

```diff
@@ -1,11 +1,16 @@
 import { getCollections, getPages, getProducts } from 'lib/shopify';
 import { MetadataRoute } from 'next';
 
+type Route = {
+  url: string;
+  lastModified: string;
+};
+
 const baseUrl = process.env.NEXT_PUBLIC_VERCEL_URL
   ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
   : 'http://localhost:3000';
 
-export default async function sitemap(): Promise<Promise<Promise<MetadataRoute.Sitemap>>> {
```

Comment:
`PromisePromisePromisePromise`

Reply:
At least it was keeping all those promises it was making. 😂